package.json

```diff
@@ -25,7 +25,7 @@
   "dependencies": {
     "babel-runtime": "7.0.0-beta.3",
     "bluebird": "2.9.25",
-    "forest-express": "5.0.0",
+    "forest-express": "5.1.0",
```

Comment:
I think we should not have more than one subject in a PR (maybe you needed this to make your PR work), even if it's hard because of the `forest-express`/`forest-express-sequelize`/`forest-express-mongoose` dependencies.

According to @arnaudbesnier, changelog have to include all the changes that were added in `forest-express` when you update a dependency, so it would have complexified this actual PR (by adding several (unrelated) lines in readme and dependency file to review)

So I think it's hard to decorrelate both subject, but merging them makes it harder to understand, review, test and git-merge too.

Maybe I'm wrong, let me know if I missed something

Reply:
The update of this is required and is the only change actually. Between `forest-express` 5.0.0 and 5.1.0 the only thing changing is the parser being called asynchronously so The changelog is good for me as the PR won't work without that change

Reply:
Thank you!